patches/FFmpeg-CVE-2020-22033.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/libavfilter/vf_vmafmotion.c b/libavfilter/vf_vmafmotion.c
```

Comment:
This is duplicate of CVE-2020-22019: 
https://nvd.nist.gov/vuln/detail/CVE-2020-22019 -> https://trac.ffmpeg.org/ticket/8241
https://nvd.nist.gov/vuln/detail/CVE-2020-22033 -> https://trac.ffmpeg.org/ticket/8246 -> duplicate of https://trac.ffmpeg.org/ticket/8241

Reply:
Yes. This path is not applied, but somehow I left it here.
Fixed.